docs/projects/cn0577/index.rst

```diff
@@ -0,0 +1,309 @@
+.. _cn0577:
+
+CN0577 HDL project
+================================================================================
+
+Overview
+-------------------------------------------------------------------------------
+
+The :adi:`EVALCN0577-FMCZ <CN0577>` provides an analog front-end and an FMC 
+digital interface for :adi:`LTC2387-18`, its core. It is a low noise, high 
+speed successive approximation register (SAR) ADC with a resolution of 18 bits 
+and sampling rate up to 15MSPS. :adi:`EVALCN0577-FMCZ <CN0577>` includes an 
+on-board reference oscillator and a retiming circuit to minimize signal-to-noise 
+ratio (SNR) degradation due to the FPGA additive jitter. In order to support 
+high speed operations while minimizing the number of data lines, a serial LVDS 
+digital interface is used. It has a one-lane and two-lane output modes, 
+allowing the user to optimize the interface data rate for each application, 
+through setting a parameter.
+Current revision of :adi:`EVALCN0577-FMCZ <CN0577>` is Rev A.
+:adi:`EVALCN0577-FMCZ <CN0577>` and :xilinx:`ZedBoard <products/boards-and-kits/1-8dyf-11.html>` 
+are connected together to build a development system setup.
+
+Supported boards
+-------------------------------------------------------------------------------
+
+-  :adi:`CN0577 <CN0577>`
+-  :adi:`LTC2387-18 chip <LTC2387-18>`
+
+Supported devices
+-------------------------------------------------------------------------------
+
+-  :adi:`ADAQ23876`
+-  :adi:`ADA4945-1`
+-  :adi:`ADG3241`
+-  :adi:`ADN4661`
+-  :adi:`ADR4520`
+
+Supported carriers
+-------------------------------------------------------------------------------
+
+.. list-table::
+   :widths: 35 35 30
+   :header-rows: 1
+
+   * - Evaluation board
+     - Carrier
+     - FMC slot
+   * - :adi:`CN0577 <CN0577>`
+     - :xilinx:`ZedBoard <products/boards-and-kits/1-8dyf-11.html>`
+     - FMC-LPC
+
+Block design
+-------------------------------------------------------------------------------
+
+The architecture is composed from one :adi:`ADA4945-1` fully differential 
```

Comment:
Not only the ADA4945-1 but also the other supported devices, right?

Reply:
I realized this info is not relevant to the HDL project. I followed the example given in the same section of CN0855's documentation before. This section will be removed.